src/components/Card.js

```diff
@@ -8,9 +8,31 @@ import {
   Text,
   Stack,
 } from '@chakra-ui/core';
-
+import URL from 'url-parse';
 import { Button, Image } from '.';
 
+function setUrlUtm(link) {
```

Comment:
Can we move this function to our `utils` folder? We may end up wanting to use this in other places as well :)

Reply:
Can do